src/main/java/com/cinema/model/ShoppingCart.java

```diff
@@ -0,0 +1,57 @@
+package com.cinema.model;
+
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "shopping_cart")
+public class ShoppingCart {
+    @Id
+    @Column(name = "cart_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    @OneToOne
+    private User user;
```

Comment:
let's use mapsId so we will not have duplications

Reply:
Done